cfme/common/__init__.py

```diff
@@ -818,3 +821,33 @@ def step(self, *args, **kwargs):
             table.click_at("Custom Button Events")
         else:
             raise DestinationNotFound("Custom button event count 0")
+
+
+class TimelinesView(View):
```

Comment:
It looks similar to `ServerTimelinesView` better to inherit and overwrite `is_displayed` method.

Reply:
Agreed, I can make this change.